### PR TITLE
feat(aerial): Add imagery christchurch-2023-0.075m to aerial config file.

### PR DESCRIPTION
**Pull request by linz-basemaps**

### CODE REVIEW

File: config/tileset/aerial.json [hunk 1]
```diff
@@ -677,12 +677,12 @@
       "category": "Urban Aerial Photos"
     },
     {
-      "2193": "s3://linz-basemaps/2193/christchurch_urban_2021_0-05m_RGB/01FPXAY15VB9MES5WQ0N2SPEG6",
-      "3857": "s3://linz-basemaps/3857/christchurch_urban_2021_0-05m_RGB/01FPXB6TVAQZV4W9KXWQ7H5ZQA",
-      "name": "christchurch-urban-2021-0.05m",
-      "minZoom": 14,
-      "title": "Christchurch 0.05m Urban Aerial Photos (2021)",
-      "category": "Urban Aerial Photos"
+      "2193": "s3://linz-basemaps/2193/christchurch_2023_0.075m/01HFART7KYYR9XX5RNHPMNRBJH/",
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:680]:
```suggestion
      "2193": "s3://linz-basemaps/2193/christchurch_2023_0.075m/01HFART7KYYR9XX5RNHPMNRBJH",
```

File: config/tileset/aerial.json [hunk 2]
```diff
@@ -677,12 +677,12 @@
...
+      "3857": "s3://linz-basemaps/3857/christchurch_2023_0.075m/01HFART7QPK7Y2TC7PY4XHHNNJ/",
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:681]:
```suggestion
      "3857": "s3://linz-basemaps/3857/christchurch_2023_0.075m/01HFART7QPK7Y2TC7PY4XHHNNJ",
```

**Comment by dwsilk** [comment]
Needs to be added at zoom level 13 (rural) as Canterbury Rural imagery over Christchurch has a bit split in the middle of it which doesn't look nice.

`christchurch-urban-2020-2021-0.075m` also needs to be removed from `aerial.json` and added as an individual tileset. The dataset that is removed at the moment only covers the Christchurch CBD.

File: config/tileset/aerial.json [hunk 3]
```diff
@@ -677,12 +677,12 @@
...
-      "name": "christchurch-urban-2021-0.05m",
-      "minZoom": 14,
-      "title": "Christchurch 0.05m Urban Aerial Photos (2021)",
-      "category": "Urban Aerial Photos"
+      "2193": "s3://linz-basemaps/2193/christchurch_2023_0.075m/01HFART7KYYR9XX5RNHPMNRBJH",
+      "3857": "s3://linz-basemaps/3857/christchurch_2023_0.075m/01HFART7QPK7Y2TC7PY4XHHNNJ",
+      "name": "christchurch-2023-0.075m",
+      "title": "Christchurch 0.075m Urban Aerial Photos (2023)",
+      "category": "Urban Aerial Photos",
+      "minZoom": 14
```
Comment (dwsilk) [inline config/tileset/aerial.json:685]:
```suggestion
      "minZoom": 13
```

**Comment by Wentao-Kuang** [comment]
> # Aerial Config Inserts
> ### christchurch-2023-0.075m
> * [NZTM2000Quad](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&i=christchurch-2023-0.075m&p=NZTM2000Quad&debug#@-43.610336,172.744797,z7) -- [Aerial](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&p=NZTM2000Quad&debug#@-43.610336,172.744797,z7)
> * [WebMercatorQuad](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&i=christchurch-2023-0.075m&p=WebMercatorQuad&debug#@-43.612217,172.743530,z9) -- [Aerial](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&p=WebMercatorQuad&debug#@-43.612217,172.743530,z9)
> 
> ### timaru-2022-2023-0.1m
> * [NZTM2000Quad](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&i=timaru-2022-2023-0.1m&p=NZTM2000Quad&debug#@-44.276473,171.205553,z7) -- [Aerial](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&p=NZTM2000Quad&debug#@-44.276473,171.205553,z7)
> * [WebMercatorQuad](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&i=timaru-2022-2023-0.1m&p=WebMercatorQuad&debug#@-44.282570,171.210937,z9) -- [Aerial](https://basemaps.linz.govt.nz/?config=s3://linz-basemaps-staging/config/config-J1uXtNyzWqnPmAVcdMZMJvUjZbyZiYktHEkftJzyCk5p.json.gz&p=WebMercatorQuad&debug#@-44.282570,171.210937,z9)

@linz/proj-basemaps-dev @dwsilk This might because the timaru-2022-2023-0.1m is merging, and rebase this before the master deployed? I will rerun the build and see if this still output two changes.

**Review by dwsilk — APPROVED** [review]

**Pull request merged**